ips_client/ips_requests.py

```diff
@@ -51,27 +45,39 @@ def post_job(self, file: IO, service: ServiceType, out_type: OutputType,
                                  timeout=settings.requests_timeout)
 
         if response.status_code != 200:
-            raise RuntimeError(f'Error while posting job: {response}')
+            raise IPSResponseError(response=response, msg=f'Error posting job')
 
         return response.json()
 
-    def get_output(self, service: ServiceType, out_type: OutputType, output_id: UUID) -> bytes:
+    @staticmethod
+    def _get_available_file_name(file: IO, file_name: Optional[str] = None) -> str:
```

Comment:
Isn't this function the exact same function in the other file? Maybe you could move them them to utils instead of implementing the same code twice?

Reply:
I was thinking the same. Be it like that! :D